perses/annihilation/new_relative.py

```diff
@@ -70,6 +75,12 @@ def __init__(self, topology_proposal, current_positions, new_positions, use_disp
         self.softcore_alpha=0.5
         self.softcore_beta=12*unit.angstrom**2
```

Comment:
I can add the units back in too, but let's only discuss the parts that changed in this PR to keep things organized. Definitely feel free to open an issue if you see a problem though!